<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChauffeurPanel</class>
 <widget class="QMainWindow" name="ChauffeurPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chauffeur Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion du trajet</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ville de depart</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ville d'arrivee</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Heure debut</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Heure fin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Poids</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Statut</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="endField"/>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="beginField"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="toField"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fromField"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="poidsField"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="statusField"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addTraButton">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editTraButton">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delTraButton">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Trajets</string>
    </property>
    <widget class="QListWidget" name="listTrajets">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>321</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Colies</string>
    </property>
    <widget class="QListWidget" name="listColis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="deliverPackageButton">
       <property name="text">
        <string>Commencer livraison</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markDeliveredButton">
       <property name="text">
        <string>Finir livraison</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
